weblate/trans/views/reports.py

```diff
@@ -83,10 +83,15 @@ def get_credits(request, path=None):
         show_form_errors(request, form)
         return redirect_param(obj or "home", "#reports")
 
+    start_date = end_date = None
+    if form.cleaned_data["period"]:
+        start_date = form.cleaned_data["period"]["start_date"]
+        end_date = form.cleaned_data["period"]["end_date"]
+
```

Comment:
Please revert the changes here as well, reports need period to be provided.

Reply:
Reverted the changes for checking if the `form.cleaned_data["period"]` is None.